src/coreComponents/finiteVolume/EmbeddedSurfaceToCellStencil.hpp

```diff
@@ -205,8 +205,12 @@ EmbeddedSurfaceToCellStencilWrapper::
   localIndex const esr1 =  m_elementSubRegionIndices[iconn][1];
   localIndex const ei1  =  m_elementIndices[iconn][1];
 
+  // DFM does not use fracture permeability but here it is used
+  // we impose t1 = t0 to be consistent with DFM implementation for comparison
+  // TODO add the contribution of fracture for both DFM and EDFM
   real64 const t0 = m_weights[iconn][0] * LvArray::tensorOps::l2Norm< 3 >( coefficient[er0][esr0][ei0][0] );
-  real64 const t1 = m_weights[iconn][1] * LvArray::tensorOps::l2Norm< 3 >( coefficient[er1][esr1][ei1][0] );
+  // real64 const t1 = m_weights[iconn][1] * LvArray::tensorOps::l2Norm< 3 >( coefficient[er1][esr1][ei1][0] );
+  real64 const t1 = t0;
```

Comment:
I suspect that this will make the impermeable cases fail.

Reply:
Do we have impermeable cases with EDFM?

Reply:
Not many, but we do have a few that test the pedfm work:
- https://github.com/GEOS-DEV/GEOS/blob/develop/inputFiles/singlePhaseFlowFractures/fractureMatrixFlow_pedfm_impermeableFracture_smoke.xml
- https://github.com/GEOS-DEV/GEOS/blob/develop/inputFiles/multiphaseFlowFractures/deadoil_3ph_corey_pedfm_impermeableFault_smoke.xml